tests/php/Dev/DeprecationTest.php

```diff
@@ -46,6 +49,10 @@ protected function setup(): void
             // Fallback to default PHP error handler
             return false;
         });
+        // This is required to clear out the message
+        // 'SilverStripe\Dev\Tests\DeprecationTest\DeprecationTestObject is deprecated. Some class message. Called from
+        // SilverStripe\ORM\Connect\TableBuilder->buildTables."
```

Comment:
```suggestion
        // This is required to clear out the notice from instantiating DeprecationTestObject in TableBuilder::buildTables().
```
We don't need to know what the full message is, especially since that could change with future updates.

Reply:
Updated